test/src/main/java/org/axonframework/test/aggregate/FixtureConfiguration.java

```diff
@@ -196,6 +197,14 @@
      */
     FixtureConfiguration<T> registerIgnoredField(Class<?> declaringClass, String fieldName);
 
+    /**
+     * Registers handler definition within this fixture.
+     *
+     * @param handlerDefinition used to create concrete handlers
+     * @return the current FixtureConfiguration, for fluent interfacing
+     */
+    FixtureConfiguration<T> registerHandlerDefinition(HandlerDefinition handlerDefinition);
```

Comment:
Is this HandlerDefinition being added to a list of definitions, or will it overwrite any previously registered one? Please make that clear in the javadoc.

Reply:
This comment has been resolved.